src/main/kotlin/Navigation/NavigationProject.kt

```diff
@@ -0,0 +1,234 @@
+                    titleTypes = TitleTypes.ARCHIVE
+                    break
+                }
+
+                val choiceTheNote = scanTheNote?.let { choiceArchive?.noteList?.elementAt(it - 1)  }
+
+                if (scanTheNote != null) {
+                    while (scanTheNote > 0) {
+                        text.printChoiceTheNoteConclusion()
+                        println("Имя заметки: ${choiceTheNote?.name}")
+                        println("Текст заметки: ${choiceTheNote?.text}")
+                        break
+                    }
+                }
+            }
+        }
+    }
+
+
+    fun startProjectArchive(): MutableList<String> {
+        val a: MutableList<String> = mutableListOf()
+        a.add(stringConcatenation(ListTextMenu.EXIT.number, ListTextMenu.EXIT.nameText))
+        a.add(
+            stringConcatenation(
+                ListTextMenu.CREATE_ARCHIVE.number,
+                ListTextMenu.CREATE_ARCHIVE.nameText
+            )
+        )
+        a.add(
+            stringConcatenation(
+                ListTextMenu.LOOK_ARCHIVE.number,
+                ListTextMenu.LOOK_ARCHIVE.nameText
+            )
+        )
+        a.add(
+            stringConcatenation(
+                ListTextMenu.CLEAR_ARCHIVE.number,
+                ListTextMenu.CLEAR_ARCHIVE.nameText
+            )
+        )
+        return a
+    }
+
+    fun startProjectTheNote(): MutableList<String> {
+        val a: MutableList<String> = mutableListOf()
+        a.add(stringConcatenation(ListTextMenu.EXIT.number, ListTextMenu.EXIT.nameText))
+        a.add(
+            stringConcatenation(
+                ListTextMenu.CREATE_THE_NOTE.number,
+                ListTextMenu.CREATE_THE_NOTE.nameText
+            )
+        )
+        a.add(
+            stringConcatenation(
+                ListTextMenu.LOOK_THE_NOTE.number,
+                ListTextMenu.LOOK_THE_NOTE.nameText
+            )
+        )
+        a.add(
+            stringConcatenation(
+                ListTextMenu.CLEAR_THE_NOTE.number,
+                ListTextMenu.CLEAR_THE_NOTE.nameText
+            )
+        )
+        return a
+    }
+
+
+    fun stringConcatenation(num: Int, text: String): String {
+        return "$num. $text"
+    }
+
+    fun printThroughLoop(print: MutableList<String>) {
+        print.let { list ->
+            list.forEach { println(it) }
+        }
+    }
+
+    fun printListArchiveMenu() {
+        printThroughLoop(startProjectArchive())
+    }
+
+    fun printListTheNoteMenu() {
+        printThroughLoop(startProjectTheNote())
+    }
+
+}
+
+enum class ListTextMenu(val number: Int, val nameText: String) {
+    EXIT(0, "Выход"),
+    CREATE_ARCHIVE(1, "Создать архив"),
+    LOOK_ARCHIVE(2, "Посмотреть архив"),
+    CLEAR_ARCHIVE(3, "Удалить архив"),
+
+    CREATE_THE_NOTE(1, "Создать заметку"),
+    LOOK_THE_NOTE(2, "Посмотреть заметки"),
+    CLEAR_THE_NOTE(3, "Удалить заметку");
+}
+
+enum class TitleTypes(val nameTitleTypes: String) {
```

Comment:
🍏 параметр не используется, можно удалить
```suggestion
enum class TitleTypes() {
```

Reply:
+